src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static String getRandomFigure() {

```

Comment:
it's better to awoid to use `static` methods, lets make `getRandomFigure()` not static